js-core-1/week-2/lesson.md

```diff
@@ -237,262 +271,135 @@ function satisfiesRequirements(num) {
-
-```sh
-$ node
-> var name = "Daniel"
-undefined
-> name.toLowerCase()
-daniel
-```
-
-Arrays also have several methods that you can use.
-
-### `.sort()`
-
-_An array method that sorts the values in an array into ascending alphabetical or numerical order._
+You can also _assign_ new values to parts of an array:
 
 ```js
-var unorderedLetters = ["z", "v", "b", "f", "g"];
-var orderedLetters = unorderedLetters.sort();
-
-var unorderedNumbers = [8, 5, 1, 4, 2];
-var orderedNumbers = unorderedNumbers.sort();
+let students = ["Ahmed", "Maria", "Atanas", "Nahidul", "Jack"]
 
-console.log(orderedLetters); // logs [ 'b', 'f', 'g', 'v', 'z' ]
-console.log(unorderedLetters); // logs [ 'b', 'f', 'g', 'v', 'z' ]
+students[2] = "Bianca"
 
-console.log(orderedNumbers); // logs [ 1, 2, 4, 5, 8 ]
-console.log(unorderedNumbers); // logs [ 1, 2, 4, 5, 8 ]
+console.log(students) // ["Ahmed", "Maria", "Atanas", "Nahidul", "Jack"]
 ```
 
-> When you call this array method it uses the array on the left side of the dot as an input, and it sorts that array also returning it. Note how both ordered and unordered arrays are sorted now!
-
-### `.concat()`
-
-_Adds (or concatenates) another value or array to the array._
+### Exercise (5 mins)
+In the node REPL, enter the following array:
 
 ```sh
-$ node
-> var arr = [1, 2, 3]
-undefined
-> arr.concat(4)
-[1, 2, 3, 4]
-> arr
-[1, 2, 3]
+> let fruits = ['banana', 'apple', 'strawberry', 'kiwi', 'fig', 'orange']
 ```
 
-Did you notice how calling the concat method did not change `arr`? This is because `concat`, like most array methods, returns a _new_ array, it does not alter the one you called the method on.
+1. Which index do you need to return:
+* strawberry
+* kiwi
+* orange
+* banana
 
-If you wan to use the array returned by calling `.concat()` you should store it in a new variable.
+2. Replace 'apple' with 'raspberry' and 'fig' with 'pineapple'
 
-```js
-var arr = [1, 2, 3];
-var newArr = arr.concat(4);
+### Exercise (5 mins)
+Complete this function so that, if the second index in array contains the name "Amy", it returns `"Second index matched!"`
 
-console.log(newArr); // logs [1, 2, 3, 4]
+```js
+function secondMatchesAmy(array) {
+  if ( ) {
+    return "Second index matched!"
+  }
+  
+  return "Second index not matched"
+}
 ```
 
-## More Array methods
-
-Let's explore some more array methods.
-
-### `.slice()`
-
-_Returns a slice of the array._
-
-You can tell `.slice()` where you want the slice to begin and end by passing it two parameters.
+## Loops
 
-```sh
-$ node
-> var arr = [0, 1, 2, 3, 4]
-undefined
-> arr.slice(0, 2)
-[0, 1]
-> ["a", "b", "c", "d"].slice(1, 2)
-['b']
-```
-
-### `.includes()`
+Loops go hand-in-hand with objects like arrays in JavaScript. A **loop**  is a way for you to go over each element in an array (or more generally, an **iterable** - we'll come to this later).
```

Comment:
I wouldn't mention the relationship between loops and arrays so explicitly. I think they need to know that loops are what they need to use for repetition. Khan Academy has a nice review:

https://www.khanacademy.org/computing/computer-programming/programming/looping/a/review-looping

Reply:
Ah, good point. See, this is what I needed to review, because I wasn't sure in what context we wanted to teach them loops (eventually the exercises will reflect that as well). Happy for the wording to be updated :)